sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_lro.py

```diff
@@ -228,6 +228,9 @@ def from_continuation_token(  # type: ignore
         continuation_token: str,
         **kwargs: Any
     ) -> "AnalyzeHealthcareEntitiesLROPoller":  # type: ignore
+        """
+        :meta private:
```

Comment:
This just hides it in the docs right? If anyone is using it for any strange reason it wont affect them?